### DISCUSSION TITLE
Not picking up schema changes to a view

### DISCUSSION THREAD

**Discussion by awisework** [TabularEditor/TabularEditor3#1399] [Data Modeling]
I have a view in snowflake (and potentially more to be created). When I go to update the schema it doesn't pick up when new columns have been added to the view. Is this a limitation of using a view? Or am I doing something wrong. this is the m code within the expression:
let
    Source = Snowflake.Databases(#"Datasource Server",#"Datasource Warehouse",[Role=#"Datasource Role"]),
    DATABASE = Source{[Name=#"Datasource DB",Kind="Database"]}[Data],
    EDW_Schema = DATABASE{[Name="EDW",Kind="Schema"]}[Data],
    Table = EDW_Schema{[Name="VIEW_NAME",Kind="View"]}[Data],

**Comment by mlonsk** [comment]
Hi @awisework 

Could you try to enable "Use Analysis Services for change detection" in the preference menu and see if that makes a difference to getting the table schema to update?

![image](https://github.com/user-attachments/assets/70519e3a-bd9b-4c64-9033-8ffdbf7e5df1)

**Reply by mlonsk** [reply]
@awisework Could you also confirm that you are using the latest version of Tabular Editor (3.18.2)

**Comment by awisework** [comment]
that seemed to do the trick, thanks! (sorry for the delayed response, was out last week)
